conference/NW/nsdi.yml

```diff
@@ -43,7 +43,7 @@
       timeline:
         - abstract_deadline: '2024-04-30 23:59:59'
           deadline: '2024-05-07 23:59:59'
-        - abstract_deadline: '2029-09-12 23:59:59'
+        - abstract_deadline: '2024-09-12 23:59:59'
           deadline: '2024-09-19 23:59:59'
       timezone: UTC-5
```

Comment:
Thanks for correcting this error. But the time zone may also have an issue: US EDT should be UTC-4 right?
per https://www.usenix.org/conference/nsdi25/call-for-papers and https://en.wikipedia.org/wiki/Eastern_Time_Zone

Reply:
Yes, you are right. I will correct it. Thanks!